blueprint/index.md

```diff
@@ -1,171 +1,176 @@
+4. Write down the access key and secret.
+5. Create an .env file in the directory folder and provide values for the following variables: `AWS_REGION`, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY`.
 
-3. **(Optional)** Use the Name box to give the app a different name.
-   ![Name Interaction Widget](images/name-interaction.png "Name Interaction Widget")
-   
-4. Click the **Configuration** tab. In the **Application URL** box, type the URL of the web application. Be sure to specify the full URL, including the https: at the beginning of the URL.
-   
-   The pcConversationId URL parameter is used to determine the conversation interaction and pcLangTag is used to determine the agent's language in the Chat Translator solution.
+  :::primary
+  **Tip**: Start with the sample.env file for this blueprint and provide your org-specific details.
+  :::
 
-   <pre class="language-nohighlight"><code class="language-nohighlight">https://localhost/?conversationid=&#123;&#123;pcConversationId&#125&#125;&language=&#123;&#123;pcLangTag&#125&#125;</code></pre>
+### Install and activate the Chat Translator in Genesys Cloud
 
-   **(Optional)** Select a group for filtering user acces to the widget.
+1. Log in to your Genesys Cloud organization and add an integration. For more information, see [Add an integration](https://help.mypurecloud.com/?p=135807 "Opens the Add an integration article") in the Genesys Cloud Resource Center.
 
-   **(Optional)** Select specific queues for widget access.
+   ![Add integration](images/add-integration.png "Add integration")
 
-   Select the ACD interaction types where you want the widget to be available for.
+2. Install the **Interaction Widget** integration. For more information, see [Set up an Interaction Widget integration](https://help.mypurecloud.com/?p=229319 "Opens the Set up an Interaction Widget integration article") in the Genesys Cloud Resource Center.
 
-   ![Interaction Configuration](images/interaction-config.png "Interaction Configuration")
-   
-5. Go to **Advanced** Configuration and enter this in the text area:
+   ![Install Interaction Widget](images/install-interaction-widget.png "Install Interaction Widget")
 
-   ```json
-   {
-     "lifecycle": {
-       "ephemeral": false,
-       "hooks": {
-         "stop": true,
-         "blur": true,
-         "focus": true,
-         "bootstrap": true
+3. (Optional) Use the **Name** box to give the widget a meaningful name. For example, **Chat Translator**.
+
+   ![Interaction Widget name](images/name-interaction.png "Interaction Widget name")
+
+4. Click the **Configuration** tab.
+5. In the **Application URL** box, type the URL of the web application. Be sure to specify the full URL, beginning with `https:`.
+
+  ```
+   https://localhost/?conversationid=&#123;&#123;pcConversationId&#125&#125;&language=&#123;&#123;pcLangTag&#125&#125;
+  ```
```

Comment:
The new Developer Center can now properly render the  double curly `{{ }}`. We may now have this replaced to `https://localhost/?conversationid={{pcConversationId}}&language={{pcLangTag}}`

Indented code fences also do not render properly in the Developer Center. We need to remove the spaces here for it to render properly.